### PR TITLE
Donut , Line Chart

### PR DESCRIPTION
**Pull request by Prakash-kumar1**

### CODE REVIEW

File: src/components/Donutchart.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      )
+      .then((response) => {
+        setMydata(response.data);
+      });
+  }, []);
+
+  const country = [];
+  const medal = [];
+  mydata.map((value) => {
+    console.log(value);
```
Comment (Souvikdeb2612) [inline src/components/Donutchart.js]: remove/comment console statement before pushing

File: src/components/Linechart.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  }, []);
+
+  const newArr = [];
+  sData.map((val) => {
+    const obj = {};
+    obj.name = val.category_title;
+    obj.data = val.value;
+    newArr.push(obj);
+  });
+  console.log(newArr);
```
Comment (Souvikdeb2612) [inline src/components/Linechart.js]: same as above

File: src/components/Linechart.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        <h2>Line Chart- Using Apexcharts in React </h2>
+        <Chart
+          type="line"
+          width={550}
+          height={450}
+          series={newArr}
+          options={{
+            title: { text: "Product sell in 2021" },
+            xaxis: {
+              categories: [
```
Comment (Souvikdeb2612) [inline src/components/Linechart.js]: store the categories in an constant and call here

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/components/Donutchart.js (hunk 1 above)
Comment (Prakash-kumar1) [reply src/components/Donutchart.js]: Done sir

File: src/components/Linechart.js (hunk 2 above)
Comment (Prakash-kumar1) [reply src/components/Linechart.js]: Done sir

File: src/components/Linechart.js (hunk 3 above)
Comment (Prakash-kumar1) [reply src/components/Linechart.js]: Done sir

File: src/components/Donutchart.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+export const Donutchart = () => {
+  const [mydata, setMydata] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(
+        `https://raw.githubusercontent.com/Prakash-kumar1/JSON/main/testModel`
+      )
+      .then((response) => {
```
Comment (Souvikdeb2612) [inline src/components/Donutchart.js:16]: use catch block

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**